plugins/hls-cabal-plugin/src/Ide/Plugin/Cabal/CabalAdd.hs

```diff
@@ -277,7 +277,7 @@ getDependencyEdit recorder env cabalFilePath buildTarget dependency = do
       logWith recorder Debug $ LogCreatedEdit edit
       pure edit
 
--- | Given a path to a haskell file, returns the closest cabal file.
+-- | Given a path to a haskell file, returns the closest cabal file, except if package.yaml is also present.
```

Comment:
Could you please expand a little why we don't return the .cabal file if a `package.yaml` file is found right nex to it?